include/edyn/util/collision_util.hpp

```diff
@@ -109,8 +109,8 @@ void process_collision(entt::entity manifold_entity, contact_manifold &manifold,
     auto [posA, ornA] = tr_view.template get<position, orientation>(manifold.body[0]);
     auto [posB, ornB] = tr_view.template get<position, orientation>(manifold.body[1]);
 
-    auto originA = origin_view.contains(manifold.body[0]) ? origin_view.get(manifold.body[0]) : static_cast<vector3>(posA);
-    auto originB = origin_view.contains(manifold.body[1]) ? origin_view.get(manifold.body[1]) : static_cast<vector3>(posB);
+    auto originA = origin_view.contains(manifold.body[0]) ? origin_view.get<edyn::origin>(manifold.body[0]) : static_cast<vector3>(posA);
```

Comment:
It is unnecessary to specify the `edyn::` namespace here and also in the other cases.